app/src/main/java/com/yterletskyi/happyfriend/features/friends/data/GlobalFriends.kt

```diff
@@ -6,4 +6,9 @@ object GlobalFriends {
         contactId = -1L,
         position = -10L,
     )
+    val MyGlobalIdea = Friend(
+        id = "my_wishlist",
+        contactId = -1L,
+        position = -10L,
```

Comment:
Setting position to `-10` means the global ideas item will have the same position as MyWishlist item. Which is not exactly what we want. Let's use something between -10 and 0 for example -5. This way a friends list will look like:
-10 (my whishlist), -5 (general ideas), 0 (some friend), 1 (some another friend), 2, 3,4 etc